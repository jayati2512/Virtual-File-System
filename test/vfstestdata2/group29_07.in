createvfs fgtvfs 1024
mountvfs sgdvfs
mountvfs fgtvfs
deletedir /melon
makedir 
makedir / orange
addfile /orange /usr/include/stdio.h
addfile /orange orn.txt /usr/include/stdio.h
deletedir /orange
deletedir /orange/orn.txt
deletedir /orange
makedir / apple
listdir /apple -R /tmp/logfl.txt
movefile /orange/orn.txt /apple
unmountvfs fgtvfs
