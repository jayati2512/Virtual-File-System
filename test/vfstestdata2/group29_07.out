createvfs_SUCCESS
mountvfs_FAILURE DATA_FILE_NOT_FOUND  
mountvfs_SUCCESS
deletedir_FAILURE CANNOT_FIND_SPECIFIED_DIR  
makedir_FAILURE VFS_INSUFFICIENT_ARGUMENTS 
makedir_SUCCESS 
addfile_FAILURE VFS_INSUFFICIENT_ARGUMENTS 
addfile_FAILURE FILE_TOO_LARGE 
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY  
deletedir_SUCCESS
deletedir_SUCCESS 
makedir_SUCCESS 
listdir_SUCCESS
movefile_FAILURE CANNOT_FIND_SOURCEFILE 
unmountvfs_SUCCESS
